src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -0,0 +1,1014 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import sas
+import sas_testcase
+from util import configurable_testcase, writeConfig, loadConfig
+
+
+class FederalGovernmentDatabaseUpdateTestcase(sas_testcase.SasTestCase):
+  """SAS federal government database update test cases"""
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_FDB_1_default_config(self, filename):
+    """Generates the WinnForum configuration for FDB.1"""
+
+    # Load device info
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Load grant request.
+    grant_g1 = json.load(
+                open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    # Set the grant frequency to overlap with the exclusion zone 'Yuma Proving Ground'
```

Comment:
Note: I'm still trying to figure out which kinds of zones are tested in this test case (I've received some conflicting information). Hopefully the FCC will give us an answer soon.